app/grants/models.py

```diff
@@ -1009,6 +1009,10 @@ def status(self):
 
     @property
     def amount_per_period_minus_gas_price(self):
+        if self.grant.pk == 86:
```

Comment:
wouldnt it be better to jsut check if `amount_per_period_to_gitcoin` is falsy?

Reply:
Oh you mean something like ?
```
if self.amount_per_period ==  self.amount_per_period_to_gitcoin:
  return self.amount_per_period
```

Reply:
that would fix it up